flatdata-cpp/test/MultiVectorTest.cpp

```diff
@@ -53,6 +53,32 @@ static auto create_view_with_3_items = []( )
     return std::make_pair( std::move( storage ), std::move( view ) );
 };
 
+static const size_t NUM_ITEM_TO_FLUSH = 32 * 1024 * 1024 / 4 + 1024; // enough to flush, and them some
+
+static auto create_storage_with_enough_items_to_flush = []( )
```

Comment:
This looks a bit weird considering we always used 'cannonical' function definitions so far. Why lambda?

Reply:
Done since it's consistent with the rest of the file. Could make sense to change all of them at some point into proper functions